cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -203,9 +198,14 @@ func (s *E2EStorageIntegration) doHealthCheck(t *testing.T) bool {
 
 	// Check if the status field in the JSON is "StatusOK"
 	if healthResponse.Status != "StatusOK" {
-		t.Logf("Received non-K status %s: %s", healthResponse.Status, string(body))
+		t.Logf("Received non-OK status %s: %s", healthResponse.Status, string(body))
 		return false
 	}
+
+	// @nocommit: this is a temporary hack to ensure reproducible failure
```

Comment:
can we remove this bit now?

Reply:
ah yes, but it was supposed to be failing CI, something's not right